README.md

```diff
@@ -12,10 +20,23 @@ These tools are included in iputils
 - rdisc
 - tracepath
 
-If you still use [old version](http://www.skbuff.net/iputils/), please consider moving forward to new releases placed here.
+## Tools removed from iputils
 
-This version also fully supports glibc, uClibc and musl-libc.
+Some obsolete tools has been removed (see
+[#363](https://github.com/iputils/iputils/issues/363)).
 
-Localization is hosted on [Fedora Weblate](https://translate.fedoraproject.org/projects/iputils/iputils/).
+| Tool | Removing commit | Latest release where included
+| ---- | --------------- | -----------------------------
```

Comment:
@nmeyerhans Latest release *where* included vs. Latest release *when* included?

Reply:
I might slightly prefer _when_ instead of _where_.

You might also just change the header to "Last release"

Reply:
"Last release" is really the best, thanks!